MH4U Dex UITests/MH4U_DexUITests.m

```diff
@@ -0,0 +1,158 @@
+    [tester tapViewWithAccessibilityLabel:GatheringSources];
+    [tester waitForViewWithAccessibilityLabel:SecretArea];
+    [tester tapRowAtIndexPath:thirteenthRow inTableViewWithAccessibilityIdentifier:MHDItemAreaSources];
+    [tester waitForViewWithAccessibilityLabel:SunkenHollow];
+    [tester tapViewWithAccessibilityLabel:HighRankItems];
+    [tester waitForViewWithAccessibilityLabel:Honey];
+}
+
+- (void)testItemMonsterSource
+{
+    [self navigateToItemEncyclopedia];
+    NSIndexPath *sixteenthRow = [NSIndexPath indexPathForRow:15 inSection:0];
+    [tester tapRowAtIndexPath:sixteenthRow inTableViewWithAccessibilityIdentifier:MHDItemEncyclopediaTableIdentifier];
+    [tester waitForViewWithAccessibilityLabel:ShinyDrops];
+    NSIndexPath *firstRow = [NSIndexPath indexPathForRow:0 inSection:0];
+    [tester tapRowAtIndexPath:firstRow inTableViewWithAccessibilityIdentifier:MHDItemMonsterSources];
+    [tester waitForViewWithAccessibilityLabel:BodyPart];
+    [tester tapViewWithAccessibilityLabel:HighRank];
+    [tester waitForViewWithAccessibilityLabel:LargeWyvernTear];
+    [tester tapViewWithAccessibilityLabel:LowRank];
+    [tester waitForAbsenceOfViewWithAccessibilityLabel:LargeWyvernTear];
+}
+
+- (void)testSmallItemFromMonsterToArea
+{
+    [self navigateToMonsterEncyclopedia];
+    NSIndexPath *firstRow = [NSIndexPath indexPathForRow:0 inSection:0];
+    [tester tapItemAtIndexPath:firstRow inCollectionViewWithAccessibilityIdentifier:MHDMonsterEncyclopediaCollectionIdentifier];
+    [tester waitForViewWithAccessibilityLabel:BodyPart];
+    [tester tapViewWithAccessibilityLabel:LowRank];
+    NSIndexPath *sixthRow = [NSIndexPath indexPathForRow:5 inSection:0];
+    [tester tapRowAtIndexPath:sixthRow inTableViewWithAccessibilityIdentifier:MHDMonsterDropsTable];
+    [tester tapViewWithAccessibilityLabel:GatheringSources];
+    [tester tapRowAtIndexPath:firstRow inTableViewWithAccessibilityIdentifier:MHDItemAreaSources];
+    [tester waitForViewWithAccessibilityLabel:AncSteppeLRItems];
+}
+
+- (void)testSmallItemFromAreaToMonster
+{
+    [self navigateToRegionEncyclopedia];
+    NSIndexPath *firstRow = [NSIndexPath indexPathForRow:0 inSection:0];
+    [tester tapItemAtIndexPath:firstRow inCollectionViewWithAccessibilityIdentifier:MHDRegionEncyclopediaCollectionIdentifier];
+    [tester tapViewWithAccessibilityLabel:HighRankItems];
+    [tester tapRowAtIndexPath:firstRow inTableViewWithAccessibilityIdentifier:MHDAreaDropsTable];
+    [tester waitForViewWithAccessibilityLabel:BlueMushroomDropSources];
+    [tester tapRowAtIndexPath:firstRow inTableViewWithAccessibilityIdentifier:MHDItemMonsterSources];
+    [tester waitForViewWithAccessibilityLabel:CongalalaOverview];
+    [tester tapViewWithAccessibilityLabel:GRank];
+    [tester waitForViewWithAccessibilityLabel:CongalalaFur];
+}
+
+- (void)testDistraction
+{
+    // This test exists to ensure that the afterEach is working as intended.
```

Comment:
Yeah.  I suppose since the other tests no longer return to the home view, it's no longer needed.  Should I remove it?